contracts/contracts/assets/rlp/src/main.sw

```diff
@@ -0,0 +1,262 @@
+// SPDX-License-Identifier: Apache-2.0
+contract;
+
+/*
+ ____  _     ____           __   ___      _     _      _                 _   
+|  _ \| |   |  _ \          \ \ / (_) ___| | __| |    / \   ___ ___  ___| |_ 
+| |_) | |   | |_) |  _____   \ V /| |/ _ \ |/ _` |   / _ \ / __/ __|/ _ \ __|
+|  _ <| |___|  __/  |_____|   | | | |  __/ | (_| |  / ___ \\__ \__ \  __/ |_ 
+|_| \_\_____|_|               |_| |_|\___|_|\__,_| /_/   \_\___/___/\___|\__|
+*/
+
+mod errors;
+mod events;
+
+use std::{
+    asset::{
+        mint_to, 
+        burn as asset_burn
+    },
+    context::*,
+    revert::require,
+    storage::{
+        storage_string::*,
+        storage_vec::*,
+    },
+    call_frames::msg_asset_id,
+    string::String
+};
+use std::hash::*;
+use helpers::{
+    utils::*, 
+    transfer::*,
+    zero::*
+};
+use src20::SRC20;
+use src3::SRC3;
+use asset::{
+    base::{
+        _total_supply as sl_total_supply,
+    },
+    supply::{
+        _burn as sl_burn,
+        _mint as sl_mint,
+    },
+};
+use asset_interfaces::rlp::RLP;
+use errors::*;
+use events::*;
+
+const DECIMALS: u8 = 9;
+const DEFAULT_SUB_ID: SubId = SubId::zero();
+
+storage {
+    gov: Identity = ZERO_IDENTITY,
```

Comment:
Add comments for all storage entities especially if not clear what they are based on name